packages/fiori/src/Wizard.ts

```diff
@@ -763,23 +786,23 @@ class Wizard extends UI5Element {
 	}
 
 	get enabledSteps() {
-		return this.slottedSteps.filter(step => !step.disabled);
+		return this.slottedSteps.filter(step => step.disabled);
 	}
 
 	get selectedStepsCount() {
 		return this.selectedSteps.length;
 	}
 
 	get slottedSteps() {
-		return this.getSlottedNodes("steps");
+		return this.getSlottedNodes("steps") as Array<WizardStep>;
```

Comment:
With #6563 we've made getSlottedNodes to be generics. Instead of casting it to `Array<WizardStep>` use can use `this.getSlottedNodes<WizardStep>("steps")`